doc/commands/repoquery.8.rst

```diff
@@ -159,7 +159,7 @@ Options
     | This is a list option.
 
 ``--whatprovides=CAPABILITY,...``
-    | Limit to packages that provide any of <capabilities>.
+    | Limit to packages that provide any of <capabilities>. When patterns :ref:`specifying file provide <file_provides_ref-label>` then it also matches file provides.
```

Comment:
The new sentence is not completely clear to me. How about something like (although I'm not great in english):
```
Capabilities :ref:`specifying a file provide <file_provides_ref-label>` are also matched against file provides.
```

Reply:
Great suggestion.